easybuild/easyconfigs/b/boto3/boto3-1.20.13-foss-2021b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'boto3'
+version = '1.20.13'
+
+homepage = 'https://github.com/loosolab/TOBIAS'
+description = """TOBIAS is a collection of command-line bioinformatics tools for
+performing footprinting analysis on ATAC-seq data, and includes: Correction of
+Tn5 insertion bias; Calculation of footprint scores within regulatory regions;
+Estimation of bound/unbound transcription factor binding sites; and
+Visualization of footprints within and across different conditions"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
```

Comment:
also, this should be at `GCCcore` level, since it doesn't have any dependencies on MPI or math libraries and is not performance sensitive